drizzle/main/0007_lazy_lady_mastermind.sql

```
@@ -0,0 +1,15 @@
+	"answer_id" uuid,
+	"type_id" uuid NOT NULL,
+	"description" text,
+	"created_by" uuid,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"updated_at" timestamp with time zone DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+ALTER TABLE "forum_reporting" ADD CONSTRAINT "forum_reporting_question_id_forum_question_id_fk" FOREIGN KEY ("question_id") REFERENCES "public"."forum_question"("id") ON DELETE cascade ON UPDATE no action;--> statement-breakpoint
+ALTER TABLE "forum_reporting" ADD CONSTRAINT "forum_reporting_answer_id_forum_answer_id_fk" FOREIGN KEY ("answer_id") REFERENCES "public"."forum_answer"("id") ON DELETE cascade ON UPDATE no action;--> statement-breakpoint
+ALTER TABLE "forum_reporting" ADD CONSTRAINT "forum_reporting_type_id_forum_reporting_type_id_fk" FOREIGN KEY ("type_id") REFERENCES "public"."forum_reporting_type"("id") ON DELETE no action ON UPDATE no action;--> statement-breakpoint
+ALTER TABLE "forum_reporting" ADD CONSTRAINT "forum_reporting_created_by_user_id_fk" FOREIGN KEY ("created_by") REFERENCES "public"."user"("id") ON DELETE no action ON UPDATE no action;
```

SQLFluff (4.1.0)

[error] Line 12, Position 97: Found unparsable section: 'FOREIGN KEY ("question_id") REFERENCES "...'

(PRS)